hw5.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+$n = 10;
+
+function Fibonachi($n)
+{
+    $Sum = 0;
+    $a = array();
+    $a [0] = 0;
+    $a [1] = 1;
```

Comment:
Це можна оголосити трохи лаконічніше =)
Якщо тобі потрібен массив з двома відомими значенями - ти можеш відразу його створити з ними
```suggestion
    $a = [0, 1];
```